src/appkit.cs

```diff
@@ -13454,8 +13490,21 @@ interface NSSearchField {
 		NSObject WeakDelegate { get; set; }
 
 		[Mac (10,11)]
+		[Advice ("No longer availabile, now a no-op")]
```

Comment:
minor: normally ends with a `.`
I thought we had a test for that... but maybe it's only on availability attributes